### ISSUE TITLE
rotation messes up drag direction

### ISSUE THREAD

**Issue by makc** [iras/3DInterface_01#1]
if you rotate the ball away, and then drag the triangle, it goes into opposite direction.

**Comment by iras** [comment]
Yes, that is correct and a known issue. Only the absolute coordinate rotation for the mouse was implemented at that stage of development.
